eve/methods/patch.py

```diff
@@ -135,8 +135,7 @@ def patch_internal(resource, payload=None, concurrency_check=False,
 
     resource_def = app.config['DOMAIN'][resource]
     schema = resource_def['schema']
-    if not skip_validation:
-        validator = app.validator(schema, resource)
+    validator = app.validator(schema, resource)
```

Comment:
Good catch, but won't `validator.document` be `None` on line 162, when `skip_validation = True`? Can't try right now but if I am correct then a better change would be to move 162 in the `else` block starting at line 157.